lib/functions/process.functions.sh

```diff
@@ -92,3 +92,45 @@ process_sh() {
         /bin/bash ${script} "$@"
     fi
 }
+
+process_trap_handle() {
+  local signal trap_list_var trap_exit_var
+  signal=$1
+  trap_list_var="_PROCESS_${signal}"
+  trap_exit_var="_PROCESS_EXIT_${signal}"
+  echo $(eval ${!trap_list_var})
```

Comment:
I'm not keen on using `echo` and `eval` here.  Is there a reason why this can't be used:

```
${!trap_list_var}
```